scripts/deploy-artifacts.sh

```diff
@@ -58,4 +58,4 @@ $PREFIX docker login -u ${DOCKER_USER_ID?} -p ${DOCKER_PASSWORD?}
 
 docker images
 
-tagAndPush "cdcservice" "eventuate-cdc-service"
+tagAndPush "eventuate-cdc-service" "eventuate-cdc-service"
```

Comment:
This should be done by gradlew composePush - not by docker push

The docker-compose file should define the images to push.

Reply:
ok, I will fix